waspc/data/Generator/templates/server/src/jobs/_jobs.js

```diff
@@ -0,0 +1,5 @@
+{{={= =}=}}
+import jobFactory from './{= jobFactoryName =}.js'
+{=& jobFnImportStatement =}
+
+export default jobFactory({= jobFnName =})
```

Comment:
So this is all logic, just creating a job via that factory with given perform function. Ok great!